builtin/plugins/dposv3/dpos.go

```diff
@@ -1521,6 +1521,9 @@ func UpdateDowntimeRecord(ctx contract.Context, downtimePeriod uint64, jailingEn
 	if ctx.FeatureEnabled(features.DPOSVersion3_4, false) {
 		jailOfflineValidator = jailingEnabled
 	}
+	if ctx.FeatureEnabled(features.DPOSVersion3_6, false) {
+		jailOfflineValidator = jailOfflineValidator && !statistic.Jailed
```

Comment:
I think it's fairly low risk to just do `if jailOfflineValidator && !statistic.Jailed` below, `emitJailEvent` doesn't change state itself, and it's highly unlikely to fail.

Reply:
true, done